README.md

```diff
@@ -56,7 +56,7 @@ Registered entry points for aiida.calculations:
 
 The example folder provides scripts to submit calculations in the calculations folder, and tutorials in jupyter notebook format in the tutorials folder.
 
-A quick demo of how to submit a calculation using the provided example files:
+A quick demo of how to submit a calculation using the provided example files (You might need create a [code](https://aiida.readthedocs.io/projects/aiida-core/en/stable/howto/run_codes.html#how-to-create-a-code) for Janus):
```

Comment:
Does `verdi presto` set up the computer, or just the profile? If it's just the profile (and maybe either way), I think it's probably best to have a separated comment mentioning the need to set up both the computer and code, rather than having this as an aside.